### ISSUE TITLE
Statistic data about header usages.

### ISSUE THREAD

**Issue by righettod** [OWASP/www-project-secure-headers#61]
### Description

Perform public to scan websites and view stats regarding these headers. Automated scanning of the top 1m sites on the web; filtering of said sites to view stats across industries and countries; published database dumps for public .

### Additional resources

* https://github.com/oshp/headers

**Comment by righettod** [comment]
Hi @riramar ,

I leveraged the resources provided by the free tier of GH for actions workflows with mermaid support to create set of little scripts to provide such statistics and update them every month automatically as well as the DB:

https://github.com/righettod/oshp-stats

I achieve to gather security headers infos for the first 200.000 entries of the Majestic Top 1 million site. I implemented an initial [collection of the global statistics](https://github.com/righettod/oshp-stats/blob/main/stats.md). 

All is documented on the project README and the repository is totally standalone.

**Comment by riramar** [comment]
That's look amazing! How about make this available under our organization (https://github.com/oshp/) and include some references under https://owasp.org/www-project-secure-headers/ before closing this task?

**Comment by righettod** [comment]
@riramar Indeed, it was the idea, can you create repo under the organization and grant me **admin** access on this new repo?

I will transfer the content and continue to work on the project 😃 

Thank you very much in advance 👍

**Comment by righettod** [comment]
Or you can grant me to create new repo on the OSHP org if you want. In this way I can transfer the repo directly:

![image](https://user-images.githubusercontent.com/1573775/159170112-4bdb3ed1-7a3c-4b4a-b617-1cb3a627a282.png)

**Comment by riramar** [comment]
I've invited you to the org. I think you'll be able to create new repos.

![image](https://user-images.githubusercontent.com/6748242/159186224-9091a81d-e439-450b-be14-6703da0f4185.png)

**Comment by righettod** [comment]
Thank you, I transferred the project to the OSHP org 😃 

![image](https://user-images.githubusercontent.com/1573775/159215087-cd9374e9-7cfd-4e4f-92ec-2de9ddb55f86.png)

I will provide a PR for the site during the week to add ref to the stat project 😃

**Comment by righettod** [comment]
Done via PR #65

**Comment by riramar** [comment]
Perfect! Thanks!

**Comment by righettod** [comment]
I have tested the both workflows to update the DB and the stats MD file and there are both functional 👍
